src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs

```diff
@@ -80,18 +80,19 @@ public static void LoadFromAssemblyName_AssemblyNotFound()
                 () => loadContext.LoadFromAssemblyName(asmName));
         }
 
-        [ActiveIssue(8563, PlatformID.AnyUnix)]
         [Fact]
         public static void LoadFromAssemblyName_ValidTrustedPlatformAssembly()
         {
             var asmName = AssemblyLoadContext.GetAssemblyName("System.Runtime.dll");
             var loadContext = new CustomTPALoadContext();
 
-            // Usage of TPA and AssemblyLoadContext is mutually exclusive, you cannot use both.
-            // Since the premise is that you either want to use the default binding mechanism (via coreclr TPA binder) 
-            // or supply your own (via AssemblyLoadContext) for your own assemblies.
-            Assert.Throws(typeof(FileLoadException), 
-                () => loadContext.LoadFromAssemblyName(asmName));
+            // We should be able to override (and thus, load) assemblies that were
+            // loaded in TPA load context.
+            var asm = loadContext.LoadFromAssemblyName(asmName);
+            Assert.NotNull(asm);
+            var loadedContext = AssemblyLoadContext.GetLoadContext(asm);
+            Assert.NotNull(loadedContext);
+            Assert.Equal(loadContext, loadedContext);
```

Comment:
Do we expect this to be the exact same object instance, or two different instances that have defined equality?  If the former, it'd be better to use Assert.Same.